pkgs/development/python-modules/solo-python/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildPythonPackage, fetchPypi
+, click, ecdsa, fido2, intelhex, pyserial, pyusb, requests}:
+
+ buildPythonPackage rec {
+  pname = "solo-python";
+  version = "0.0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12s5lv2n02an8nscspv8s1gknh593l7mqljqvxpnbkzfjf6icfvc";
+  };
+
+  propagatedBuildInputs = [
+    click
+    ecdsa
+    fido2 
+    intelhex
+    pyserial
+    pyusb
+    requests
+  ];
+
+  # test failes with:
+  #
+  # running build_ext
+  # Usage: nix_run_setup [OPTIONS] COMMAND [ARGS]...
+  # Try "nix_run_setup --help" for help.
+  #
+  # Error: No such command "test".
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python tool and library for SoloKeys";
+    homepage = https://github.com/solokeys/solo-python;
```

Comment:
RFC 45
```suggestion
    homepage = "https://github.com/solokeys/solo-python";
```

Reply:
Hah, I did this a few years a ago and my reviewer asked me to remove the double ticks :laughing: